src/twisted/trial/newsfragments/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
I'd normally use .gitkeep for this, is this a Twisted idiom?

Reply:
I don't know. I just copied what I found in ` src/twisted/newsfragments/`

```
$ cat src/twisted/newsfragments/.gitignore
!.gitignore
```

Reply:
Cool. lgtm